src/components/line-tabs/styles/SLineTabs.tsx

```diff
@@ -0,0 +1,55 @@
+import styled, { css } from 'styled-components';
+import { CSSProgressiveBody03Semibold } from '../../../constants/styles/fonts';
+import { MIN_SILVER, MIN_GOLD, MIN_PLATINUM, MIN_DIAMOND } from '../../../constants/styles/mediaquerys';
+import { ILineTabsProps } from '../LineTabs';
+
+const CSSHover = css`
+  color: var(--hover-text-04);
+  border-bottom: 1px solid var(--hover-text-04);
```

Comment:
@steiningermona please add a pointer cursor to have visual feedback on mouse over.